### PR TITLE
exploring the design a little bit more, avoiding calculators

### PR DESCRIPTION
**Pull request by alexandreaquiles**
I tried to avoid calculators and classes that are just wrappers to double values.

### CODE REVIEW

**Comment by alexandreaquiles** [comment]
Sorry. I didn't test if is compiling correctly, as I don't have Android Studio and my IntelliJ didn't work correctly.

File: beeHealthy/src/main/java/br/com/devcapu/bee_healthy/domain/user_info/Objective.java [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
+
     LOSE(-400),
     GAIN(300),
     DEFINE(0);
 
-    private final int calorieRatio;
+    public final int calorieRatio;
 
-    Objective(int calorieRatio) {
+    public Objective(int calorieRatio) {
```
Comment (DevCapu) [inline beeHealthy/src/main/java/br/com/devcapu/bee_healthy/domain/user_info/Objective.java:11]: Esse `public` não compila

**Review by DevCapu — CHANGES_REQUESTED** [review]
Opa, só deixei uns comentários.

OBS: Nos modelos está faltando o import de `UserInfo`.

File: beeHealthy/src/main/java/br/com/devcapu/bee_healthy/domain/health/BasalEnergyExpenditure.java [hunk 2]
```diff
@@ -6,8 +6,13 @@
...
+    public BasalEnergyExpenditure(UserInfo userInfo) {
+        if (userInfo.weight <= 0) {
+            throw new IllegalArgumentException("Weight should be greater than 0");
+        }
+
+        BasalEnergyExpenditureCoefficientsByGender coefficients = from(gen);
+        this.value = (coefficients.coefficient1 * userInfo.weight) + (coefficients.coefficient2 * (userInfo.height * 100)) - (coefficients.coefficient3 * userInfo.age) + coefficients.coefficient4;
     }
 
     public double getValue() {
```
Comment (DevCapu) [inline beeHealthy/src/main/java/br/com/devcapu/bee_healthy/domain/health/BasalEnergyExpenditure.java:18]: Se a gente deixar o atributo value como public igual nas outras classes podemos remover esse método.

**Review by DevCapu — APPROVED** [review]

**Pull request merged**